release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2237,13 +2237,13 @@ In the following tables, features are marked with the following statuses:
 
 |Configure the `br-ex` bridge needed by OVN-Kuberenetes using NMState
```

Comment:
Unrelated to this change, but I just noticed there's a typo in this line: s/OVN-Kuberenetes/OVN-Kubernetes/

Reply:
Good catch, updated.